src/server/node_manager.c

```diff
@@ -1459,6 +1459,14 @@ set_vnode_state(struct pbsnode *pnode, unsigned long state_bits, enum vnode_stat
 	}
 
 	if (pnode->nd_state & INUSE_PROV) {
+#if MOM_ALPS
+		if (!(pnode->nd_state & VNODE_UNAVAILABLE) ||	
+			(pnode->nd_state == INUSE_PROV)) { /* INUSE_FREE is 0 */	
+			DBPRT(("%s: calling [is_vnode_prov_done] from set_vnode_state, type = %d\n", id, type))	
+			is_vnode_prov_done(pnode->nd_name);	
+		}
+#endif /* MOM_ALPS */
```

Comment:
The server has never had a need to have code ifdef'd for ALPS before.  All of the ALPS code is in the MoM, hence the name "MOM_ALPS".   Please find a more generic way to add the code you need.

Reply:
If we can not use this macro. I will look for the other way to find alps mom.